src/common/reducers/snaps.js

```diff
@@ -10,7 +10,9 @@ export function snaps(state = {
     case ActionTypes.FETCH_SNAPS:
       return {
         ...state,
-        isFetching: true
+        isFetching: true,
+        success: false,
+        error: null
```

Comment:
Yeah, that was exactly my train of thought. :-)  I think it's best to keep the existing data for now, but we may decide differently in the future.